input/write_reactors.py

```diff
@@ -184,7 +283,7 @@ def region_render(array, template, full_template, output_file):
         os.system('rm ' + country + '_region')
 
 
-def main(csv_file, reactor_template, region_template,
+def main(csv_file, reactor_template, deployinst_template,
```

Comment:
Obviously, the name of this variable is important for the user, but doesn't actually (by itself) impact the functionality of the code. However, changing the name indicates to me that you feel that this function is now only appropriate for modifying the deployinst template. Why?

What if someone put the name of the region template jinja file here, where it used to go, what would happen? Will it work, or will later logic in this script no longer work for the region template? Will anyone ever want to do the region template version of this again? Why or why not?